### PR TITLE
Multi bracket validation

### PR DESCRIPTION
**Pull request by mparkersimms**
- [ x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ x] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: utilities/app/src/test/java/utilities/MultiBracketValidationTester.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package utilities;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class MultiBracketValidationTester {
```
Comment (Mattpet26) [inline utilities/app/src/test/java/utilities/MultiBracketValidationTester.java:6]: Good work with your tests! It's good to have multiple tests for all the cases you might run into. Does it return false if the input given is an empty string?
